addons/pf-maint.pl

```diff
@@ -252,12 +282,9 @@ sub print_dot {
 }
 
 sub accept_binary_patching {
-    print "=" x 110 . "\n";
     print "Should we patch the Golang binaries? ".join(",", @patchable_binaries)." Any custom code in them will be overwritten y/n [y]: ";
```

Comment:
We need to make "Any custom code in them will be overwritten" more visible.

Reply:
Not sure how we can make it more visible other than by putting it just before the [y/n]
Unless we use some fancy bash coloring

Reply:
Agree with these two options.

Reply:
Its already just before the [y/n] prompt